docs/misc/markdown.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 5 minutes Markdown guide
+date: 2017-12-21
+author: [Dima German]
```

Comment:
Don't forget to add yourself to this dict - https://github.com/datahq/frontend/blob/d50d3ec6451459678490dfbc00049cd86b8e903f/routes/index.js#L206-L212

Reply:
done